src/app/body/board/sprint-details/sprint-details.component.html

```diff
@@ -1,55 +1,55 @@
-                            <div class="row py-3">
-                                <input class="col-md-8 col-8" style="font-size: 11px;" type="number" id="filterSprintNumberInput" placeholder="enter sprint number" [(ngModel)]="filterSprintNumber">
-                                <div class="col-md-4 col-4 pl-2">
-                                    <button class="btn" id="dropdownButton" style="font-size: 14px;" (click)="changeSprintNumber()">Filter</button>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div class=" col-md-2 col-2 ">
-                    <button class="btn " id="button" type="button" style="font-size: 14px;" (click)="completeSprint()">Completed</button>
-                    <span class="material-icons align-middle" *ngIf="Status === 'Completed' ">
-                        check_circle
-                    </span>
-                </div>
-            </div>
-        </div>
-        <div class="card-body" style="background-color:#202124;">
-            <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Details: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintDetail" style="font-size: 12px;"></h6>
-            </div>
-            <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Start Date:
-                </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStartDate" style="font-size: 12px;">
-                    {{StartDate}}</h6>
-            </div>
-            <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">End Date: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintEndDate" style="font-size: 12px;">{{EndDate}}
-                </h6>
-            </div>
-            <div class="row py-2">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Status: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{Status}}</h6>
-            </div>
-        </div>
-    </div>
+<div class="pt-3 text-white">
+    <div class="card" style="border: none;">
+        <div class="card-header" style="font-weight: bold; background-color: #5A20F0; color:white;">
+            <div class="row  align-items-center">
+                <div class="col-md-2 col-4">
+                    <h5 style="font-size: 18px;">Sprint No:</h5>
+                </div>
+                <div class="col-md-7 col-5">
+                    <h5 id="sprintId">{{currentSprintName}}</h5>
+                </div>
+
+                <div class="row col-md-3 col-8 text-right">
+                    <div class="mr-1">

```

Comment:
just for margin a div tag with class, we can give padding once column is implemented.